BraveWallet/WalletLoadingButton.swift

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+
+struct WalletLoadingButton<Title: View>: View {
+  @Binding var isLoading: Bool
+  var action: () -> Void
+  var title: Title
+  
+  @ScaledMetric private var length: CGFloat = 20.0
+  
+  init(
+    isLoading: Binding<Bool>,
```

Comment:
This does not need to be a `Binding` as you are not setting `isLoading` from within the view